src/components/selectorComponents/TextInput.tsx

```diff
@@ -1,18 +1,43 @@
-import React from "react";
+import React, { useState } from "react";
 import styles from "./TextInput.module.css";
 import classNames from "classnames";
 
 interface TextInputProps {
-  placeholder?: string;
+  placeholder: string;
 }
 
 const TextInput = (props: TextInputProps) => {
+  const [currentValue, setCurrentValue] = useState("");
+  const [error, setError] = useState(false)
   const wrapStyle = classNames(styles.wrapper);
   const inputStyle = classNames(styles.input);
 
+
+  //same way i handled this in Fischkapp
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const newValue = event.target.value;
+    if (newValue.length > 35) {
+      setError(true);
```

Comment:
**issue:** We should not hardcode such validation inside `handleChange` in generic component like `TextInput`. You can leave this validation for now, as you will return to this when implementing form logic